app/ai/voice/agents/breeze_buddy/features/summarizer/context_summarizer.py

```
@@ -0,0 +1,237 @@
+            # Separate system messages from conversation messages
+            system_messages = [msg for msg in self._messages if msg["role"] == "system"]
+            conversation_messages = [
+                msg
+                for msg in self._messages
+                if msg["role"] in ["user", "assistant", "tool"]
+            ]
+            if len(conversation_messages) < self._keep_recent_turns * 2:
+                return
+
+            # Log context BEFORE summarization
+            logger.info(f"=== BREEZE BUDDY SUMMARIZATION START ===")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)